jenkins/hourly_maintenance.py

```diff
@@ -137,13 +140,41 @@ def KillLoopingBash():
             err |= subprocess.call(['sudo', 'kill', '-9', pid])
     return err
 
+
+def DeleteCorruptGitRepos():
```

Comment:
For reference, only a single build occurred between the worker going offline and the next run of hourly maintenance for the most recent flake.

This means you'd only expect one or two extra flakes per week from the gap. I think that's not _too_ bad, versus the extra work to make this run on boot or before every build.